code/modules/projectiles/gun.dm

```diff
@@ -473,11 +484,15 @@
 	if(user == target)
 		if(!ishuman(user))	// Borg suicide needs a refactor for this to work.
 			return
-		target.visible_message("<span class='warning'>[user] sticks [src] in [user.p_their()] mouth, ready to pull the trigger...</span>", \
-		"<span class='userdanger'>You stick [src] in your mouth, ready to pull the trigger...</span>")
+		target.visible_message(
+			"<span class='warning'>[user] sticks [src] in [user.p_their()] mouth, ready to pull the trigger...</span>",
+			"<span class='userdanger'>You stick [src] in your mouth, ready to pull the trigger...</span>"
+			)
 	else
-		target.visible_message("<span class='warning'>[user] points [src] at [target]'s head, ready to pull the trigger...</span>", \
-			"<span class='userdanger'>[user] points [src] at your head, ready to pull the trigger...</span>")
+		target.visible_message(
```

Comment:
maybe adding a message here that tells the user that they feel something cold and metallic pressed against the side of their head?

Reply:
The way it's set up, the blind person will actually get the second message as they are the target.